server/gamenode/Lobby.ts

```diff
@@ -51,7 +52,55 @@ export class Lobby {
 
         if (this.game) {
             this.sendGameState(this.game);
+        } else {
+            this.sendDeckInfo();
+        }
+    }
+
+    private updateDeck(socket, command, ...args) {
+        const activeUser = this.users.find((u) => u.id === socket.user.username);
+        const userDeck = activeUser.deck.data;
+        const [cardID] = args; // ['cardID']
+
+        // Determine the arrays we are moving between
+        const sourceArray = command === 'Deck' ? userDeck.deckCards : userDeck.sideboard;
```

Comment:
Is there anywhere that declares the legal values for `command`? Ideally there'd be an enum or something. Also, feels like we should probably have a check here to validate that it's not something unexpected (or null).